Deployment_Procedure.md

```diff
@@ -49,73 +51,244 @@ Create `eosio.forum`, set contract, and set permission
+eosc -u https://bos.eoshenzhen.io:9443 system newaccount eosio escrow.bos --stake-net "10.0000 BOS" --stake-cpu "10.0000 BOS" --transfer --buy-ram-kbytes 100 --skip-sign --expiration 604800 --write-transaction create_escrowbos.json --auth-filer escrow.bos.yml
 
+eosc -u https://bos.eoshenzhen.io:9443 system newaccount eosio eosio.forum --stake-net "10.0000 BOS" --stake-cpu "10.0000 BOS" --transfer --buy-ram-kbytes 100 --skip-sign --expiration 604800 --write-transaction create_eosioforum.json --auth-file eosio.forum.yml
 
-cleos -u https://bos.eoshenzhen.io:9443 system newaccount eosio eosio.forum EOS5Qqi6f4atV3sXWBxK6MsK53ELJ2cnbo3KZRnu54X3kgdJgR6u2 --stake-cpu "10.0000 BOS" --stake-net "10.0000 BOS" --buy-ram "20.0000 BOS" -p eosio -s -j -d > newacct_eosioforum.js
+eosc -u https://bos.eoshenzhen.io:9443 system newaccount eosio bet.bos --stake-net "10.0000 BOS" --stake-cpu "10.0000 BOS" --transfer --buy-ram-kbytes 100 --skip-sign --expiration 604800 --write-transaction create_auditorbos.json --auth-file auditor.bos.yml
 ```
 
 
 
 ### 2.2 multisig propose_trx
 
-```
-cleos -u https://bos.eoshenzhen.io:9443 multisig propose_trx createbetbos bppermission.json newacct_betbos.json pursonchen22 
-
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig propose_trx createescrow bppermission.json  newacct_escrowbos.json pursonchen22  
+```shell
+eosc -u https://bos.eoshenzhen.io:9443 multisig propose pursonchen22 createbetbos create_betbos.json --request-producers
 
+eosc -u https://bos.eoshenzhen.io:9443 multisig propose pursonchen22 createescrow create_escrowbos.json --request-producers
 
-cleos -u https://bos.eoshenzhen.io:9443 multisig propose_trx createaudito  bppermission.json  newacct_auditorbfos.json pursonchen22  
+eosc -u https://bos.eoshenzhen.io:9443 multisig propose pursonchen22 createforum create_eosioforum.json --request-producers
 
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig propose_trx 
-createforum bppermission.json  newacct_eosioforum.json pursonchen
+eosc -u https://bos.eoshenzhen.io:9443 multisig propose pursonchen22 createaudito create_auditorbos.json --request-producers
 ```
 
 
 
-
-
 ### 2.3 BP APPROVE
 
-```
-cleos -u https://bos.eoshenzhen.io:9443 multisig approve createbetbos '{"actor":"BP_name","permission":"active"}' -p BP_name
-
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig approve createaudito '{"actor":"BP_name","permission":"active"}' -p BP_name
-
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig approve createforum '{"actor":"BP_name","permission":"active"}' -p BP_name
-
-
-cleos -u https://bos.eoshenzhen.io:9443 multisig approve createescrow '{"actor":"BP_name","permission":"active"}' -p BP_na
+```shell
+eosc multisig approve pursonchen22 createbetbos <BP ACCOUNT>
```

Comment:
either use `bosc` or use the `-u` BOS endpoint flag

Reply:
agree with you